experimentation/tools/achievements/README.md

```diff
@@ -0,0 +1,26 @@
+# Achievements script
+This is a script that can be used to generate an "achievements" file for
+Sorald, detailing the pull requests performed with it. It's essentally
+a human-readable version of the PRs JSON file.
+
+## Requirements
+Requires Python 3.8+
+
+## Install
+Install the packages listed in `requirements.txt`.
+
+```bash
+$ python3 -m pip install -r requirements.txt
+```
+
+> **Note:** You should do this in a virtual environment to avoid polluting
+> system-wide or user packages.
```

Comment:
I believe it can be useful to give some basic commands here, e.g.:

```bash
pip install virtualenv
virtualenv --python=python3.6 env
source ./env/bin/activate
```

WDYT?

Reply:
I agree, but I think it would be better left for #326 where I bring all of the scripts together, as these particular instructions currently exist in several of the script directories (yay, duplication).

Reply:
Understood and agreed!